src/test/java/io/aiven/kafka/connect/common/output/JsonLinesOutputWriterTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2020 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.common.output;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+import org.apache.kafka.common.record.TimestampType;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.sink.SinkRecord;
+
+import io.aiven.kafka.connect.common.output.jsonwriter.JsonLinesOutputWriter;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+
+public class JsonLinesOutputWriterTest {
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    protected ByteArrayOutputStream byteStream;
+    protected static JsonLinesOutputWriter sut;
```

Comment:
I'd recommend making these variables local to each test where they're used. No immediate problem with them, but having them as fields opens the way to tests starting affecting each other due to mistakes.

Reply:
Sorry, don't get this point. Isn't a  'beforeEach' statement just made for this purpose?

Reply:
It is.
Well, I see your point. Looks a little bit strange to me, but 🤷